config/i18n/all-langs.ts

```diff
@@ -223,19 +224,19 @@ export enum LangCodes {
  */
 export const hiddenLangs = ['arabic'];
 
-// locale is sourced from a JSON file, so we use getLangCode and getLangName to
+/**
+ * This array contains languages that use the RTL layouts.
+ */
+export const rtlLangs = ['arabic', 'urdu'];
+
+// locale is sourced from a JSON file, so we use getLangCode to
 // find the associated enum values
 
 export function getLangCode(locale: PropertyKey) {
   if (isPropertyOf(LangCodes, locale)) return LangCodes[locale];
   throw new Error(`${String(locale)} is not a valid locale`);
 }
 
-export function getLangName(locale: PropertyKey) {
-  if (isPropertyOf(LangNames, locale)) return LangNames[locale];
-  throw new Error(`${String(locale)} is not a valid locale`);
-}
-
```

Comment:
it wasn't used, so I removed it. You can check the commit and its tests here https://github.com/freeCodeCamp/freeCodeCamp/pull/48245/commits/b68a30a33e08f7e1a4479207103b03b0e2bc06a4